retrieval/scrape.go

```diff
@@ -285,13 +285,15 @@ type targetScraper struct {
 }
 
 const acceptHeader = `application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited;q=0.7,text/plain;version=0.0.4;q=0.3,*/*;q=0.1`
+const userAgentHeader = `Prometheus-scraper/1.0`
```

Comment:
I'm not sure we'd remember to update this with each logic handling change, and there's a moderate chance of an unintentional change.